test/tests/performance/pbs_qsub_performance.py

```diff
@@ -53,30 +52,27 @@ def setUp(self):
 
     def test_submit_job(self):
         """
-         submission of 1000 jobs
-         before and after exporting variable
-         to check submission performance
```

Comment:
This explains what the test is doing, but not why.  Change this docstring to: "Test qsub without the -V option.  The existence of huge environmental variables should not affect submission time.  First submit 1000 jobs and time it.  Add a huge environmental variable and submit 1000 more jobs.  The two sets of jobs should take about the same amount of time to submit."